tools/protobuf-compile/protobuf-compile.go

```diff
@@ -104,7 +104,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	protocGenGoGrpcExec, err := filepath.Abs(protocGenGoExec)
+	protocGenGoGrpcExec, err = filepath.Abs(protocGenGoGrpcExec)
```

Comment:
This is the other part of the bug fix mentioned [here](https://github.com/hashicorp/terraform-plugin-go/pull/570/files#r2380207829)